README.md

```diff
@@ -34,60 +36,43 @@
+### Custom CSS ([Vencord](https://vencord.dev/), [BetterDiscord](https://betterdiscord.app/))
 
-1. Simply add your preferred flavour into your discord clients CustomCSS file/editor.
+Add `@import url("https://catppuccin.github.io/discord/dist/catppuccin-<flavor>-<accent>.theme.css");` to your Discord client's custom CSS file/editor, where `<flavor` and `<accent>` are your preferred flavor and accent, respectively. To simply go with the default accent color, omit the `-<accent>` part. For example:
 
 ```css
-/* latte */
-@import url("https://catppuccin.github.io/discord/dist/catppuccin-latte.theme.css");
-/* frappe */
-@import url("https://catppuccin.github.io/discord/dist/catppuccin-frappe.theme.css");
-/* macchiato */
-@import url("https://catppuccin.github.io/discord/dist/catppuccin-macchiato.theme.css");
-/* mocha */
+/* Mocha, with the default accent: */
 @import url("https://catppuccin.github.io/discord/dist/catppuccin-mocha.theme.css");
 
-/* You can also append Catppuccin colors to customize the accent, e.g. */
-/* mocha (pink accent)*/
+/* Mocha, with a custom pink accent: */
 @import url("https://catppuccin.github.io/discord/dist/catppuccin-mocha-pink.theme.css");
-/* frappe (maroon accent) */
-@import url("https://catppuccin.github.io/discord/dist/catppuccin-frappe-maroon.theme.css");
+
+/* Of course, you can do the same with any flavor and any accent - not just mocha or pink: */
+@import url("https://catppuccin.github.io/discord/dist/catppuccin-latte.theme.css");
+@import url("https://catppuccin.github.io/discord/dist/catppuccin-latte-blue.theme.css");
 ```
 
-### [Stylus](https://github.com/openstyles/stylus)
+> [!TIP]
+> You can also have Discord automatically switch themes depending on your system theme preference. The following snippet showcases a configuration that switches between latte in light mode and mocha in dark mode by adding an inline [`prefers-color-scheme` media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme), `(prefers-color-scheme: <light-or-dark>)`, after each `@import` statement (see ["Importing CSS rules conditional on media queries" - MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@import#importing_css_rules_conditional_on_media_queries)).
+>
+>  ```css
+>  @import url("https://catppuccin.github.io/discord/dist/catppuccin-mocha.theme.css")
+>  (prefers-color-scheme: dark);
+>  @import url("https://catppuccin.github.io/discord/dist/catppuccin-latte.theme.css")
+>  (prefers-color-scheme: light);
+>  ```
+
+### [Stylus](https://github.com/openstyles/stylus) (Web)
 
 1. Enable CSP Patching from Stylus Settings > Advanced.
-2. [Click here to install](https://github.com/catppuccin/discord/raw/main/discord.user.css).
-3. Choose your preferred flavour and accent color from the Stylus preference dropdown.
+2. Visit the following URL to bring up the Stylus install page for the Discord userstyle: https://github.com/catppuccin/discord/raw/main/discord.user.css.
```

Comment:
I'm not a huge fan of the language change here, is there any reason to do this?

Reply:
More clarity about what you are clicking on, why it would install it.